locale/en/blog/weekly-updates/weekly-update.2016-01-29.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Weekly Update - Jan 29th, 2016
+author: Minwoo Jung (@jmwsoft)
+date: 2016-01-29T12:00:00.000Z
+status: publish
+category: weekly
+slug: weekly-update-2016-01-29
+layout: blog-post.hbs
+---
+
+### Node.js News
+
+The Node.js project will be releasing new versions across all of its active release lines early next week (possibly sooner, pending full impact assessment) to incorporate upstream patches from OpenSSL and some additional low-severity fixes relating to HTTP handling. Please read on for full details.
+
+See https://nodejs.org/en/blog/vulnerability/openssl-and-low-severity-fixes-jan-2016/ for more information.
+
+### OpenSSL Impact Assessment
+
+OpenSSL versions 1.0.1r and 1.0.21 have been released, the announcement can be found here: https://mta.openssl.org/pipermail/openssl-announce/2016-January/000061.html
+
+Our team has made an assessment of the impact of the disclosed defects and concluded that there is no urgency in releasing patched versions of Node.js in response to this release. Therefore, we will be proceeding as planned and attempt to release new versions of each of our active release lines on or before
```

Comment:
I think this was correct originally. Should be `on or after`. See https://github.com/nodejs/nodejs.org/blob/b3469dc2c4f1a45a5ac9ac09c9b7f02c5e42951e/locale/en/blog/vulnerability/openssl-and-low-severity-fixes-jan-2016.md#update-29-jan-2016-openssl-impact-assessment